tests/factories/cht/generate.js

```diff
@@ -83,26 +122,52 @@ const createClinic = (index, healthCenter) => {
   return { clinic, primaryContact };
 };
 
-const createAdditionalPersons = (nbrPersons, clinic) => {
+const createAdditionalPersons = (nbrPersons, clinic, ageType) => {
   return Array
-    .from({ length: nbrPersons - 1 })
+    .from({ length: nbrPersons })
     .map((_, i) => {
-      const additionalPersonName = `${FIRST_NAMES[i % FIRST_NAMES.length]} ${LAST_NAMES[i % LAST_NAMES.length]}`;
       const additionalPhoneNumber = PHONE_NUMBERS[i % PHONE_NUMBERS.length];
+      let name;
+      let additionalPersonName;
+      let date_of_birth;
+      if (ageType === 'kid') {
+        name = ADDITIONAL_KID_FIRST_NAMES[i % ADDITIONAL_KID_FIRST_NAMES.length];
+        additionalPersonName = `${name} ${clinic.last_name}`;
+        date_of_birth = DATE_OF_BIRTHS_KIDS[i % DATE_OF_BIRTHS_KIDS.length];
+      } else if (ageType === 'adult') {
+        name = ADDITIONAL_WOMAN_FIRST_NAMES[i % ADDITIONAL_WOMAN_FIRST_NAMES.length];
+        additionalPersonName = `${name} ${clinic.last_name}`;
+        date_of_birth = DATE_OF_BIRTHS_ADULTS[i % DATE_OF_BIRTHS_ADULTS.length];
+      }
+
       return personFactory.build({
         parent: { _id: clinic._id, parent: clinic.parent },
         name: additionalPersonName,
```

Comment:
Here it can be calculated in the return
```suggestion
        name: `${name} ${clinic.last_name}`,
```